### PR TITLE
feat: use selection colors from theme

### PR DESCRIPTION
**Pull request by haxxmaxx**
Enabling selection colors from dataColors. This is behind feature flag

### CODE REVIEW

File: packages/sn-filter-pane/src/hooks/use-styling/selections-style.ts [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
 
-  const selected = getColor(selectionColors.selected || {}, false) || '#009845';
-  const alternative = getColor(selectionColors.alternative || {}, false) || '#E4E4E4';
-  const excluded = getColor(selectionColors.excluded || {}, false) || '#A9A9A9';
-  const selectedExcluded = getColor(selectionColors.selectedExcluded || {}, false) || '#A9A9A9';
-  const possible = getColor(selectionColors.possible || {}, false) || '#FFFFFF';
+export default function getSelectionsStyle({ componentsOverrides, stardustTheme, getListboxStyle }: IGetSelectionStyles) {
+  const getColor = (type: SelectionType) => {
+    const componentColor = componentsOverrides?.selections?.colors?.[type];
+    const evaluatedColorFromComponents = componentColor && stardustTheme?.getColorPickerColor ? stardustTheme.getColorPickerColor(componentColor, false) : undefined;
```
Comment (haxxmaxx) [inline packages/sn-filter-pane/src/hooks/use-styling/selections-style.ts:29]: idk if stardustTheme is ever undefined, but kept the sanity check

**Comment by haxxmaxx** [comment]
There is something strange going on with the formatting in this repo. Seems like prettier wants to break at 80 characters, I assume it is not setup with the common configs?

**Review by DanielS-Qlik — APPROVED** [review]

File: packages/sn-filter-pane/src/hooks/use-styling/selections-style.ts [hunk 2]
```diff
@@ -1,28 +1,50 @@
...
 
-  const getColor = stardustTheme?.getColorPickerColor || (() => false);
+export default function getSelectionsStyle({
+  componentsOverrides,
+  stardustTheme,
+  getListboxStyle,
+  env,
+}: IGetSelectionStyles) {
+  // eslint-disable-next-line no-confusing-arrow
+  const getThemeDataColor = (type: SelectionType) => env?.flags?.isEnabled('PS_22149_THEME_SELECTION_COLORS') ? getListboxStyle('', `dataColors.${type}`) : undefined;
```
Comment (DanielS-Qlik) [inline packages/sn-filter-pane/src/hooks/use-styling/selections-style.ts:35]:
```suggestion
  const getThemeDataColor = (type: SelectionType) => {
    const isEnabled = env?.flags?.isEnabled('PS_22149_THEME_SELECTION_COLORS');
    return isEnabled ? getListboxStyle('', `dataColors.${type}`) : undefined;
  };
```

**Review by DanielS-Qlik — APPROVED** [review]

File: packages/sn-filter-pane/src/hooks/use-styling/selections-style.ts (hunk 2 above)
Comment (haxxmaxx) [reply packages/sn-filter-pane/src/hooks/use-styling/selections-style.ts:35]: agree, that is cleaner

**Pull request merged**